### ISSUE TITLE
Segmentation fault detected when running ddrgen

### ISSUE THREAD

**Issue by ChengJin01** [eclipse/omr#2885]
Segmentation fault occurred in ddrgen when compiling the latest version of OpenJDK11/OpenJ9/OMR with VS2017.

```
../ddrgen --blob ../j9ddr.dat --superset ../superset.dat --blacklist blacklist --macrolist ../mac    roList --overrides overrides --show-empty  \
        ../j9ddr_misc29.pdb ../j9gc29.pdb ../j9jvmti29.pdb ../j9prt29.pdb ../j9shr29.pdb ../j9thr    29.pdb ../j9trc29.pdb ../j9vm29.pdb ../jclse11_29.pdb
make[5]: *** [run_omrddrgen.mk:73: ../j9ddr.dat] Segmentation fault
make[5]: Leaving directory '/cygdrive/c/temp1/openj9-openjdk-jdk11/build/windows-x86_64-normal-se    rver-release/vm/ddr'
```
It worked good last month after fixing all issues with OpenJDK11 on Windows.

I am not too sure whether the problem was caused by recent changes merged in omr/ddr

@keithc-ca,  @DanHeidinga

**Comment by keithc-ca** [comment]
I was able to reproduce this. It fails because it can't resolve typedefs like `omr_error_t` immediately.
The `PostponedType` mechanism handles delayed resolution of field types, but `PdbScanner::createTypedef` also want to copy the type size into the typedef. Perhaps another (later) pass can update sizes of typedefs.

**Comment by keithc-ca** [comment]
@ChengJin01 This should be fixed by #2944: please close if you agree.
